### PR TITLE
homework_6 was added

### PR DESCRIPTION
**Pull request by Denis-Yeromenko**

### CODE REVIEW

File: homeworks/denis.yeromenko_Denis-Yeromenko/homework_6/homework_6_1.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const user = {};
+
+Object.defineProperty(user, 'fullName', {
+    get: () => this.fullName,
```
Comment (markoffden) [inline homeworks/denis.yeromenko_Denis-Yeromenko/homework_6/homework_6_1.js]: arrow functions here will access global scope, avoid this

File: homeworks/denis.yeromenko_Denis-Yeromenko/homework_6/homework_6_1.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    set: (value) => {
+        let clearName = value.trim().toLowerCase().split(/\s+/);
+        for (let i = 0; i < clearName.length; i++) {
+            clearName[i] = clearName[i].charAt(0).toUpperCase() + clearName[i].slice(1);
+        }
+        clearName = clearName.join(' ').split(/-/);
+        for (let i = 0; i < clearName.length; i++) {
+            clearName[i] = clearName[i].charAt(0).toUpperCase() + clearName[i].slice(1);
+        }
+        this.fullName = clearName.join('-');
```
Comment (markoffden) [inline homeworks/denis.yeromenko_Denis-Yeromenko/homework_6/homework_6_1.js]: very complicated, look at `\b\w` for example

File: homeworks/denis.yeromenko_Denis-Yeromenko/homework_6/homework_6_1.js (hunk 1 above)
Comment (Denis-Yeromenko) [reply homeworks/denis.yeromenko_Denis-Yeromenko/homework_6/homework_6_1.js]: fixed

File: homeworks/denis.yeromenko_Denis-Yeromenko/homework_6/homework_6_1.js (hunk 2 above)
Comment (Denis-Yeromenko) [reply homeworks/denis.yeromenko_Denis-Yeromenko/homework_6/homework_6_1.js]: fixed

**Pull request merged**